integration-tests/src/lib.rs

```diff
@@ -237,7 +237,7 @@ where
     let config = config::load_with_args([0u8; 0], args);
 
     start::<A>(config)
-        .instrument(info_span!("test", test_id = %services.id))
+        .instrument(error_span!("test", test_id = %services.id))
```

Comment:
it's not logging an error,

it' adding a span (i.e. information which will be included when logging an event) with the error log level

the reason we do so is because we want the span always be enabled, e.g. if it is `info` and we only log errors when logging an error it would not include the span

I did the same change in the draft PR I did to do some additional testing for the multi tenant code.

The reason we do add the test_id as a span is to know to which tests logs belong, but it just works soso as spans don't span across spawned futures.